docs(readme): improve readme (#11)
sapphire-project/utilities: packages/snowflake/README.md

<div align="center">

![Sapphire Logo](https://cdn.skyra.pw/gh-assets/sapphire.png)

# @sapphire/snowflake

> Deconstructs and generates snowflake IDs using BigInts</h4>

Reviewer: ```suggestion
Deconstruct and generate snowflake IDs using BigInts.</h4>
```

-   [Buy us some doughnuts](#buy-us-some-doughnuts)
-   [Contributors ✨](#contributors-%E2%9C%A8)

## Description

> There is often a need to get a unique ID for entities, be that for Discord messages/channels/servers, keys in a database or many other similar examples. There are many ways to get such a unique ID, and one of those is using a so-called "snowflake". You can read more about snowflake IDs in [this Medium article].(https://medium.com/better-programming/uuid-generation-snowflake-identifiers-unique-2aed8b1771bc)

Reviewer 2: ```suggestion
There is often a need to get a unique ID for entities, be that for Discord messages/channels/servers, keys in a database or many other similar examples. There are many ways to get such a unique ID, and one of those is using a so-called "snowflake". You can read more about snowflake IDs in [this Medium article](https://medium.com/better-programming/uuid-generation-snowflake-identifiers-unique-2aed8b1771bc).
```

> There is often a need to get a unique ID for some entry, be that for Discord messages / channels / servers etc. or just keys in a database or many other similar examples. There are many ways to get such a unique ID, and one of those is using a so-called "snowflake". You can read more about Snowflake IDs in [this medium article](https://medium.com/better-programming/uuid-generation-snowflake-identifiers-unique-2aed8b1771bc)

Reviewer: ```suggestion
There is often a need to get a unique ID for entities, be that for Discord messages/channels/servers, keys in a database or many other similar examples. There are many ways to get such a unique ID, and one of those is using a so-called "snowflake". You can read more about snowflake IDs in [this Medium article].(https://medium.com/better-programming/uuid-generation-snowflake-identifiers-unique-2aed8b1771bc)
```

## API Documentation

For the full API documentation please refer to the TypeDoc generated [documentation](https://sapphire-project.github.io/utilities/modules/_sapphire_snowflake.html).

## Buy us some doughnuts

> Sapphire Project is and always will be open source, even if we don't get donations. That being said, we know there are amazing people who may still want to donate just to show their appreciation. Thanks you very much in advance!

Reviewer 2: ```suggestion
Sapphire Project is and always will be open source, even if we don't get donations. That being said, we know there are amazing people who may still want to donate just to show their appreciation. Thank you very much in advance!
```
